pom.xml

```diff
@@ -108,7 +109,7 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
+            <!-- plugin>
```

Comment:
I use this to cross-reference existing dependencies against reported dependency version vulnerabilities in the National Vulnerability Database. Please add this back in.